tests/network/rdp/test_rdp_for_exposed_vm_svc.py

```diff
@@ -75,6 +75,7 @@ def rdp_pod(workers_utility_pods, rdp_vm):
     ],
     indirect=True,
 )
+@pytest.mark.xfail(reason="CNV-43997", run=False)
```

Comment:
It looks like this mark will cause the test to always fail with the reason "CNV-43997".
Did you perhaps mean the following? 
```suggestion
@pytest.mark.xfail(reason="CNV-43997", run=False)
```

Reply:
@RoniKishner I don't see the difference between my change and your suggestion.
You probably mean `mark.skip`, right?
Anyway - you are partially correct :) : The test will indeed fail, but won't be marked as fail, but as skipped.
But anyway, skip seems like a better option, because it will save the precious time spent on attempting to execute this test and fail.

Reply:
```
@pytest.mark.xfail(reason="CNV-43997", run=False)
```
is the right approach, the test wil not be executed thus not marked as failed.
The reason should be more explanation; e.g `CNV-43997 - client side fix....`